PRODUCTION_DEPENDENCIES.md

```diff
@@ -30,6 +31,7 @@
 | [chartjs-plugin-datalabels](https://github.com/chartjs/chartjs-plugin-datalabels) | 2.0.0 | MIT |
 | [date-fns-tz](https://github.com/marnusw/date-fns-tz) | 1.3.7 | MIT |
 | [date-fns](https://github.com/date-fns/date-fns) | 2.29.2 | MIT |
+| [energinet-datahub](undefined) | 0.0.0 | MIT |
```

Comment:
issue: something went wrong here

Reply:
What should it be?

Reply:
Should it be excluded, because this is what is in our package.json

Reply:
It's added because the `private` property was removed from our `package.json`. Any reason for that?

Reply:
package.json should be private